### DISCUSSION TITLE
How to crop the extension

### DISCUSSION THREAD

**Discussion by pingbaohua** [eclipse-theia/theia#14060] [Q&A]
How can I log out the left Explorer Search Debug programs and keep only the plugins I want to install
The top menu only registerMneu "Run"

![image](https://github.com/user-attachments/assets/37136067-62b3-46f2-aacd-2d6678b0b205)

**Comment by msujew** [comment]
Hey @pingbaohua,

you can disable contributions from Theia by using the `ContributionFilter` API. See [our documentation](https://theia-ide.org/docs/contribution_filter/). You can find the specific contribution you need to disable [here](https://github.com/eclipse-theia/theia/blob/master/packages/search-in-workspace/src/browser/search-in-workspace-frontend-contribution.ts).

**Comment by pingbaohua** [comment]
Thanks, I commented out the following code and Search is disbabled
theia/packages/search-in-workspace/src/browser/search-in-workspace-frontend-module.ts
` // bind(SearchInWorkspaceFactory).toSelf().inSingletonScope();
    // bind(WidgetFactory).toService(SearchInWorkspaceFactory);`
